uuid.js

```diff
@@ -11,6 +11,9 @@
   // returns 128-bits of randomness, since that's what's usually required
   var _rng;
 
+  // Allow for MSIE11 msCrypto
+  var _crypto = _global.crypto || _global.msCrypto;
```

Comment:
Sure, but in the (very unlikely) case of both, or where someone else has defined crypto globally, do you prefer the non-standard name over the standard name?

I can say without any question that MSIE11 does not support "crypto" natively, but only supports msCrypto. (crypto was in MSIE10.)